app.py

```diff
@@ -16,8 +16,18 @@
 from fastapi import Depends, FastAPI, File, HTTPException, Request, UploadFile
 from fastapi.responses import RedirectResponse, StreamingResponse, JSONResponse
 from pydantic import BaseModel
+from fastapi.middleware.cors import CORSMiddleware
 
 app = FastAPI()
+origins = ["*"]
```

Comment:
We don't want do add this on the code layer.

If we add this, please only set a wildcard allow origin header using a feature flag.